cypress/integration/requests/requests-sort.spec.js

```diff
@@ -0,0 +1,69 @@
+import testType from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import Requests from '../../support/fragments/requests/requests';
+import EditRequest from '../../support/fragments/requests/edit-request';
+import { MultiColumnListHeader } from '../../../interactors';
+
+describe('ui-requests: Sort requests', () => {
+  const usersData = [];
+  const requestsData = [];
+  const cancellationReasons = [];
+
+  beforeEach(() => {
+    cy.login(Cypress.env('diku_login'), Cypress.env('diku_password'));
+    cy.getToken('diku_admin', 'admin');
+
+    Requests.requestTypes.forEach((requestType) => {
+      const itemStatus = requestType === 'Page' ? 'Available' : 'Checked out';
+      Requests.createRequestApi(itemStatus, requestType).then(({
+        createdUser,
+        createdRequest,
+        cancellationReasonId
+      }) => {
+        usersData.push(createdUser);
+        requestsData.push(createdRequest);
+        cancellationReasons.push(cancellationReasonId);
+      });
+    });
+  });
+
+  afterEach(() => {
+    Requests.requestTypes.forEach((_, i) => {
```

Comment:
@jakhongirabdukhamdiov Let's delete created `request`, `instance`, `holding`, `item`  all together instead of cancel `request`. Take a look at this [FAT-887](https://github.com/folio-org/stripes-testing/pull/311) for more info.